libr/core/cmd_anal.c

```diff
@@ -47,6 +47,7 @@ static void var_help(RCore *core, char ch) {
 		"afvg", " [idx] [addr]", "define var get reference",
 		"afvs", " [idx] [addr]", "define var set reference",
 		"afx", "[-] [from] [to]", "manipulate function xrefs",
+		"afM", "", "print byte map of functions coverage",
```

Comment:
Yes, we already discusses about moving the commands (including aai) into ai.
However, I think that should be a different pr as it may affect several commands.